### PR TITLE
Add an EFS volume to the assessment environment

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

This PR adds an EFS volume to the assessment environment.

## 💭 Motivation and Context

The assessors require persistent storage in their environments.

## 🧪 Testing

All pre-commit hooks pass.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by felddy — APPROVED** [review]
Neat! 🌟

**Review by mcdonnnj — APPROVED** [review]
This is the kind of persistence I can get behind. 🎸

File: private_acl_rules.tf [hunk 1]
```diff
@@ -81,6 +81,9 @@ resource "aws_network_acl_rule" "private_egress_to_operations_via_ssh" {
 # Allow ingress from operations subnet via ephemeral ports
 # For: DevOps ssh access from private subnet to operations subnet and
 #      Assessment team VNC access from private subnet to operations subnet
+#
+# Note that this also covers ingress from the operations subnet via
+# TCP port 2049 for EFS.
```
Comment (dav3r) [inline private_acl_rules.tf:86]: 👍 Excellent comment, thanks for adding it.

**Review by dav3r — APPROVED** [review]
Looks great! 💾

**Pull request merged**